src/main/java/mate/academy/model/service/ConnnectionService.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.model.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnnectionService {

```

Comment:
this seems to be a complete duplicate of `ConnectionUtil` - why do you need it?

Reply:
Oh, I forgot to delete it after I moved it to appropriate place...